setup.cfg

```diff
@@ -18,6 +18,7 @@ addopts =
   -p no:logging
   --color=yes
   --doctest-glob=*.rst
+  --doctest-modules
```

Comment:
@Kiy4h you'll have to delete this line for the builds to pass. A pull request has already been made to solve this issue https://github.com/timofurrer/ramlient/pull/62. For not just undo this change.